pyproject.toml

```diff
@@ -45,10 +45,9 @@ classifiers = [
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
 dependencies = [
-    "argschema >= 3.0.4",
     "bcbio-gff == 0.7.0",
     "biopython == 1.81",
-    "ensembl-hive @ git+https://github.com/Ensembl/ensembl-hive.git",
+    "ensembl-hive @ git+https://github.com/Ensembl/ensembl-hive.git",  # minimum v1.2.0
```

Comment:
no dependency on `https://github.com/Ensembl/ensembl-py` added

Reply:
we have it in the dev section, should it be moved from there?

Reply:
:facepalm: Changed now. Cheers!